great-idea/css/index.css

```diff
@@ -145,11 +145,12 @@ section.cta div h1 {
 /* Main Content Selectors (index.html) */
 
 section.main-content {
-	background-image: url("C:/Users/kingmoc/Documents/git/User-Interface/great-idea-website/img/mid-page-accent.jpg");
+	background-image: url("../img/mid-page-accent.jpg");
```

Comment:
again, this is a small error in this path that is preventing your images from being displayed

Reply:
This is interesting in that when I tried running this from the live server - it wouldn't work using only 1 dot.  Very interesting.